lib/src/cupertino_progress_bar.dart

```diff
@@ -25,29 +25,10 @@ class CupertinoVideoProgressBar extends StatefulWidget {
 }
 
 class _VideoProgressBarState extends State<CupertinoVideoProgressBar> {
-  _VideoProgressBarState() {
-    listener = () {
-      setState(() {});
-    };
-  }
-
-  VoidCallback listener;
   bool _controllerWasPlaying = false;
 
   VideoPlayerController get controller => widget.controller;
 
-  @override
-  void initState() {
-    super.initState();
-    controller.addListener(listener);
```

Comment:
Why is the listener stuff not needed anymore?

Reply:
This was never used (and incorrectly never unregistered).

Reply:
@Ahmadre or @erickok since I am not sure :)